src/shared_modules/utils/zlibHelper.hpp

```diff
@@ -88,6 +97,126 @@ namespace Utils
             } while (len == sizeof(buf));
             outputFile.close();
         }
+
+        /**
+         * @brief Uncompress ZIP file and returns a list with the decompressed files.
+         *
+         * @param zipFilePath Compressed (.zip) file path.
+         * @param outputDir Folder where the output files will be stored.
+         * @return std::vector<std::string> List of decompressed files.
+         */
+        static std::vector<std::string> zipDecompress(const std::filesystem::path& zipFilePath,
```

Comment:
Very interesting. Couldn't reproduce it though.

I added a check. But instead of
```c++
if (outputFilepath.string().rfind(outputDir.string(), 0) != 0)
```
I used
```c++
if (!Utils::startsWith(outputFilepath.string(), outputDir.string()))
```